.github/workflows/latex-lint.yml

```
@@ -0,0 +1,55 @@
+name: LaTeX Lint
+
+on:
+  push:
+    branches: [ main, master ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)